app/js/app.js

```diff
@@ -72,3 +72,9 @@ appendChildController.mainController = mainController;
 moveController.mainController = mainController;
 
 window.mainController = mainController;
+
+// We insert the main view into the document on startup.  This adds a
+// small disabled stylesheet and a minimal offscreen element to the
+// DOM. The mainView keeps the rest of the customizer out of the DOM
+// until its open() method is called.
+document.body.appendChild(mainView.el);
```

Comment:
Why is this needed? We should avoid appending views outside control logic if at all possible. Thus, the ideal flow should be:
```app.js -> RoutingController > MainController > MainView```